packages/alfa-rules/src/common/dom/get-offset-parent.ts

```diff
@@ -6,7 +6,9 @@ import { Predicate } from "@siteimprove/alfa-predicate";
 const { and, or, not, test } = Predicate;
 const { hasName, isElement } = Element;
 
-import { isPositioned, isRendered, isRoot } from "../predicate";
+import { isPositioned } from "../predicate/is-positioned";
+import { isRendered } from "../predicate/is-rendered";
+import { isRoot } from "../predicate/is-root";
```

Comment:
How come these imports are no longer grouped together?

Reply:
I want to avoid the risk of circular imports. Some predicates depend on DOM helpers (e.g. currnetly `hasInterposedDescendants` depends on `getOffsetParent`, and some DOM helpers depend on predicate (as here). By importing from `predicate.ts` we almost create an import loop; it seems to be OK by some JS magic only importing the bare minimum… But it felt incomfortable and I prefered breaking the loop from the start 😅